lib/cli/input/init.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require('inquirer');
+
+const questions = [
+{
+  name: 'username',
+  type: 'input',
+  message: 'Enter your Autolab username:',
+  validate: function(value) {
+    if (value.length) {
+      return true;
+    }
+    else {
+      return 'Please enter your username';
+    }
+  }
+},
+{
+  name: 'password',
+  type: 'password',
+  message: 'Enter your password:',
+  validate: function(value) {
+    if (value.length) {
+      return true;
+    }
+    else {
+      return 'Please enter your password';
+    }
+  }
+},
+];
+
+const getInput = (args, options) => {
+  if (typeof options.u == 'string' && typeof options.p == 'string')
```

Comment:
Instead of reducing the work to a low-level logical conditions, it might be better to have a function named `areCredentialsValid()` which would wrap the two conditions for now. The advantage of putting it in a function is to retain the flexibility to have policies on username and password strings later on.